src/sprout/Controllers/Admin/ManagedAdminController.php

```diff
@@ -611,7 +619,7 @@ public function _importData($filename)
                 }
             }
 
-            if (!$match_csv and $_POST['duplicates'] != 'new') {
+            if ((empty($match_db) or empty($match_csv)) and $_POST['duplicates'] != 'new') {
```

Comment:
Funny, I'd say prefer to keep using the `!$match_csv` style statement, especially now you've got the null declarations.

`empty()` behaves like `isset()` and skips over undefined variables and I'm starting to avoid it now. Because you'd never know if you've made a typo like `empty($merch_csv)` - it would just work (or not) but there's no error.